setup.py

```diff
@@ -28,7 +28,7 @@
                       ],
     extras_require={
         "GUI": ["PyQt5==5.12.1", "PyQt5-stubs==5.12.1"],
-        "TEST": ["selenium"],
+        "TEST": ["selenium", "pytest"],
```

Comment:
Technically, this is not needed for tests to run, but it does allow for a developer to run a single test manually after a suite fails so it's a nice QOL improvement.